src/docs/automation-and-resiliency/application-resource-tuning.md

```diff
@@ -0,0 +1,285 @@
+
+### Viewing quota usage
+
+You can identify current resource quota consumption and properly size resource requests and limits of existing/new workloads using either the OpenShift web console or `oc` command-line tool.
+
+#### Viewing quota usage (GUI)
+
+From the OpenShift web console, in the **Administrator** perspective, proceed to **Administration** \> **ResourceQuotas** and select the appropriate `ResourceQuota` (i.e., `compute-long-running-quota`). Here is an example:
+
+![tools example compute-long-running-quota](https://github.com/BCDevOps/developer-experience/blob/master/docs/images/tools-example-compute-long-running-quota.png?raw=true)
+
+#### Viewing quota usage (CLI)
+
+To describe a specific quota, use the `oc` tool:
+
+```console 
+$ oc describe resourcequotas compute-long-running-quota # -n <project>
+Name:       compute-long-running-quota
+Namespace:  <license>-tools
+Scopes:     NotBestEffort, NotTerminating
+ * Matches all pods that have at least one resource requirement set. These pods have a burstable or guaranteed quality of service.
+ * Matches all pods that do not have an active deadline. These pods usually include long running pods whose container command is not expected to terminate.
+Resource         Used    Hard
+--------         ----    ----
+limits.cpu       1860m   8
+limits.memory    5184Mi  32Gi
+pods             9       100
+requests.cpu     610m    4
+requests.memory  2752Mi  16Gi
+```
+
+### Risks reducing resource reservation
+
+Consider these risks when reducing resource quotas (and subsequently, requests/limits).
+
+See [Configuring your cluster to place pods on overcommitted nodes](https://docs.openshift.com/container-platform/latest/nodes/clusters/nodes-cluster-overcommit.html) for more details.
+
+#### Simultaneous resource claiming
+
+Reducing resource requests ["works on the assumption that not all the pods will claim all of their usable resources at the same time"](https://cloud.redhat.com/blog/full-cluster-part-2-protecting-nodes).
+
+Consideration must be made to determine if several workloads across the cluster would be bursting above their requests, simultaneously at a specific time of day.
+
+#### Node CPU saturation
+
+Very low CPU requests (i.e., `5m`) may be assigned to workloads such as Jenkins that have minuscule CPU usage when idle, and rely on CPU limits to burst during pipeline runs. A potential risk with this configuration is if the node a workload is scheduled on is being heavily utilized, the workload will not be able to burst much higher than the given CPU requests, potentially causing a significant slowdown.
+
+However, this will not cause pod evictions, and CPU throttling (extensively below CPU limits) can be mitigated ensuring nodes across the cluster are evenly balanced and not over-utilized.
```

Comment:
```suggestion
However, this will not cause pod evictions. CPU throttling (extensively below CPU limits) can be mitigated, ensuring nodes across the cluster are evenly balanced and not over-utilized.
```